includes/Plugin.php

```diff
@@ -210,6 +210,7 @@ function() use ( $options, $activation_flag ) {
 				( new Core\Feature_Tours\Feature_Tours( $this->context, $user_options ) )->register();
 				( new Core\Util\Migration_1_3_0( $this->context, $options, $user_options ) )->register();
 				( new Core\Util\Migration_1_8_1( $this->context, $options, $user_options, $authentication ) )->register();
+				( new Core\Util\Migration_1_119_0( $this->context, $modules->get_module( 'analytics-4' ), $options, $user_options ) )->register();
```

Comment:
It's instantiated here to follow the other two classes, and for decoupling it from `Analytics_4` class, although it is needed as dependency.  I think it is more appropriate to go here, then in `Analytics_4` class, but let me know (CR/MR reviewer) if you think differently.

Reply:
Update: Changed the approach to use `Analytics_Settings` instead and avoided passing the `Analytics_4` instance